client/state/i18n/locale-suggestions/actions.js

```diff
@@ -0,0 +1,26 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { I18N_LOCALE_SUGGESTIONS_ADD, I18N_LOCALE_SUGGESTIONS_REQUEST } from 'state/action-types';
+
+/**
+ * Action creator function: I18N_LOCALE_SUGGESTIONS_ADD
+ *
+ * @param {Object} items - list of locale suggestions
+ * @return {Object} action object
+ */
+export const receiveLocaleSuggestions = items => ( {
+	type: I18N_LOCALE_SUGGESTIONS_ADD,
+	items,
+} );
+
+/**
+ * Action creator to request locale suggestions: I18N_LOCALE_SUGGESTIONS_ADD
```

Comment:
`s/I18N_LOCALE_SUGGESTIONS_ADD/I18N_LOCALE_SUGGESTIONS_REQUEST/`

Reply:
Whoops, overlooked that comment. Thanks @deBhal. I'll update it in the related PR #21900